build.gradle

```diff
@@ -88,7 +88,7 @@ subprojects {
     testImplementation testLibraries.hamcrest
     testImplementation testLibraries.awaitility
     testImplementation testLibraries.osgiCompile
-    testImplementation platforms.collect { platform(it) }
+    testImplementation testPlatforms.collect { platform(it) }
```

Comment:
I think this would be both platforms + testPlatforms? Since now the constraints are not applied universally like before, I am unsure if the separation is as important. But I do think it would be additive rather than exclusive, at least.

Reply:
Fixed! In my projects, I tend to enable consistent version resolution, but I shouldn't assume that here.